SwiftUIDZ/SwiftUIDZ/String+Extension.swift

```diff
@@ -0,0 +1,37 @@
+// String+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Расширение для стринги с форматирование номера телефона
+extension String {
+    func makeNumberString() -> String {
+        replacingOccurrences(of: "[^0-9]", with: "", options: .regularExpression)
+    }
+
+    func removeAllSpace() -> String {
+        replacingOccurrences(of: " ", with: "")
+    }
+
+    func formatPhone(with mask: String) -> String {
+        let numbers = removeAllSpace().makeNumberString()
+        var result = ""
+        var index = numbers.startIndex
+        for ch in mask.enumerated() where index < numbers.endIndex {
+            if ch.element == "X" {
```

Comment:
```suggestion
//Mark: - public methods
    func makeNumberString() -> String {
        replacingOccurrences(of: "[^0-9]", with: "", options: .regularExpression)
    }

    func removeAllSpace() -> String {
        replacingOccurrences(of: " ", with: "")
    }

    func formatPhone(with mask: String) -> String {
        let numbers = removeAllSpace().makeNumberString()
        var result = ""
        var index = numbers.startIndex
        for ch in mask.enumerated() where index < numbers.endIndex {
            if ch.element == "X" {
```
марки и стринги в константы